### ISSUE TITLE
the configuration about your pre-train model

### ISSUE THREAD

**Issue by mnoiy** [xinntao/ESRGAN#79]
hello,thanks for you wanderful work,this project helps me a lot,
I want to train the model to fit my own scales size, actually my picture is low quality but the same size with the HR pictures,so I must modify your net and thus I can not use  your pre-train model --RRDN_PSNR_x4 ，so I  want to know the train details about your pre-train model like iterations and learning rate,etc.
